<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DepositWindow</class>
 <widget class="QDialog" name="DepositWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Deposit calculator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
	 background-color: rgb(236, 240, 241);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="11" column="1" colspan="2">
    <widget class="QRadioButton" name="caps_button">
     <property name="styleSheet">
      <string notr="true">QRadioButton {
  font-size: 16px;
  color: black;
}

</string>
     </property>
     <property name="text">
      <string>—Åapitalization of interest</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QDateEdit" name="input_date">
     <property name="styleSheet">
      <string notr="true">QDateEdit {
   color: black;
   font-size: 16px;
   background-color : white;
}
</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="result_show_2">
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
  font-size: 16px;
  color: black;
}

</string>
     </property>
     <property name="text">
      <string> placement period</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="period_input">
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
   color: black;
   font-size: 16px;
   background-color : white;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="17" column="0">
    <widget class="QLabel" name="result_show_14">
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
  font-size: 16px;
  color: black;
}

</string>
     </property>
     <property name="text">
      <string> tax</string>
     </property>
    </widget>
   </item>
   <item row="19" column="0">
    <widget class="QLabel" name="result_show_13">
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
  font-size: 16px;
  color: black;
}

</string>
     </property>
     <property name="text">
      <string> total deposit</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="result_show_4">
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
  font-size: 16px;
  color: black;
}

</string>
     </property>
     <property name="text">
      <string> tax rate</string>
     </property>
    </widget>
   </item>
   <item row="17" column="1">
    <widget class="QLabel" name="tax_show">
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{   
  color: black;
   font-size: 16px;
   background-color : white;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="14" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="18" column="1">
    <spacer name="verticalSpacer_10">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="months">
     <property name="styleSheet">
      <string notr="true">QLabel {
  font-size: 16px;
  color: black;
}
</string>
     </property>
     <property name="text">
      <string>months</string>
     </property>
    </widget>
   </item>
   <item row="20" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="result_show_3">
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
  font-size: 16px;
  color: black;
}

</string>
     </property>
     <property name="text">
      <string> interest rate</string>
     </property>
    </widget>
   </item>
   <item row="13" column="1" colspan="2">
    <widget class="QComboBox" name="freq_of_pay">
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox{
  color: black;
  font-size: 15px;
  background-color: white;
}
</string>
     </property>
     <item>
      <property name="text">
       <string>daily</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>monthly</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>quarterly</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>every six months</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>yearly</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>at the end of the period</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="15" column="0">
    <widget class="QLabel" name="result_show_15">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
  font-size: 16px;
  color: black;
}

</string>
     </property>
     <property name="text">
      <string> income</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <spacer name="verticalSpacer_7">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="2">
    <widget class="QLabel" name="sign_2">
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
  font-size: 16px;
  color: black;
}

</string>
     </property>
     <property name="text">
      <string>%</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLineEdit" name="interest_rate_input">
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
   color: black;
   font-size: 16px;
   background-color : white;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <spacer name="verticalSpacer_6">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="9" column="2">
    <widget class="QLabel" name="sign">
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
  font-size: 16px;
  color: black;
}

</string>
     </property>
     <property name="text">
      <string>%</string>
     </property>
    </widget>
   </item>
   <item row="21" column="1">
    <widget class="QPushButton" name="pushButton_calculate">
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
  font-size: 16px;
  color: white;
  background-color: rgb(255, 157, 3);
}
QPushButton::pressed{
  background-color: rgb(244, 129, 2);
}</string>
     </property>
     <property name="text">
      <string>calculate</string>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <widget class="QLabel" name="result_show_9">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
  font-size: 16px;
  color: black;
}

</string>
     </property>
     <property name="text">
      <string> frequency of payments</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">QLabel {
  font-size: 16px;
  color: black;
}

</string>
     </property>
     <property name="text">
      <string> start of term</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QLineEdit" name="tax_rate_input">
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
   color: black;
   font-size: 16px;
   background-color : white;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="result_show">
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
  font-size: 16px;
  color: black;
}

</string>
     </property>
     <property name="text">
      <string> amount of deposit</string>
     </property>
    </widget>
   </item>
   <item row="19" column="1">
    <widget class="QLabel" name="total_deposit">
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{   
  color: black;
   font-size: 16px;
   background-color : white;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="16" column="1">
    <spacer name="verticalSpacer_9">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <spacer name="verticalSpacer_5">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QLineEdit" name="sum_input">
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
   color: black;
   font-size: 16px;
   background-color : white;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
   </item>
   <item row="15" column="1">
    <widget class="QLabel" name="income_show">
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{   
  color: black;
   font-size: 16px;
   background-color : white;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <spacer name="verticalSpacer_8">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
